README.md

```diff
@@ -22,49 +34,149 @@ You can do it manually following the instructions from [rust-lang.org](https://w
 
 Otherwise, CMake would not be able to compile Rust-related targets and will produce a warning about that during configuration.
 
-## Clone
+### CodeStyle
 
-To clone repository execute
-```
-git clone https://github.com/filecoin-project/cpp-filecoin.git
-cd cpp-filecoin
-git submodule update --init --recursive
-```
+We follow [CppCoreGuidelines](https://github.com/isocpp/CppCoreGuidelines).
+
+Please use clang-format 11.0.0 with provided [.clang-format](.clang-format) file to autoformat the code.
+
+## Maintenance
+
+Maintainers: @zuiris, @turuslan, @Elestrias, @ortyomka, @wer1st, @Alexey-N-Chernyshov
+
+Tickets: Can be opened in Github Issues.
+
+
+# Cpp-Filecoin install and setup guide
+## Fuhon node establishing
+There is 2 ways of getting executable version of filecoin node and miner. The First is to get executables by downloading, you can do it here: `https://link-tofiles`.The second is to build executables by yourself, see **Advanced building**. To begin running a node you will need `node.exe`
```

Comment:
maybe comment this section until link is available

Reply:
`.exe` extension is too windows-specific, use executable.

Reply:
fixed

Reply:
link is still missing

Reply:
no release -> no link